### ISSUE TITLE
How to add a custom watermark for each pdf pages without creating a new one in memory with MuhammaraJS

### ISSUE THREAD

**Issue by natnaeltibebu** [julianhille/MuhammaraJS#297]
Hey, please help me! I want to add a watermark to each page only on 'input.pdf' and save it.

**Comment by julianhille** [comment]
And you don't want to have the file in memory?
Single pages in memory is ok?
Please elaborate a bit further as im not sure i understood correctly

**Comment by natnaeltibebu** [comment]
Yes, I want it to add on itself; a single page is okay if possible.
